exampleSite/config.yaml

```diff
@@ -11,6 +11,10 @@ params:
   # Enable float footnotes.
   # Default to true
   enableFootnotes: true
+  sitename: "Hugo Tania is Amazing"
```

Comment:
I prefer to use Camel-Case.
In this case, it will be `siteName`.

Reply:
I've already updated to Camel-Case